security/.gitignore

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
What is the purpose of this top level folder?

Reply:
For Security ADRs like best way to implement hardware root of trust abstraction underpinning secret store.